zerver/lib/push_notifications.py

```diff
@@ -914,6 +925,11 @@ def handle_push_notification(user_profile_id: int, missed_message: Dict[str, Any
         PushDeviceToken.objects.filter(user=user_profile, kind=PushDeviceToken.APNS)
     )
 
+    logger.info(
+        "Sending mobile push notifications for user %s: via FCM %s devices, via APNs %s devices",
```

Comment:
We might want to tweak these strings to be more parseable; @alexmv may have thoughts on how best to do that.

Reply:
They'd read more naturally to me as:
```
Sending mobile push notifications for user %s: %d via FCM devices, %d via APNs devices
```